static/styles/zulip.css

```diff
@@ -280,6 +280,7 @@ p.n-margin {
     position: relative;
 
     .alert {
+        display: block;
```

Comment:
Why do you make this change?  This sort of detail should be explained in the commit message; I assume the theory is that because we're only keeping active alerts in the DOM, we no longer need to do show/hide logic?

Reply:
Yeah, I missed it!
Also instead of adding this, I should have removed,
https://github.com/zulip/zulip/blob/467028511cb59eadab8a6be53f0b4e5024bcacf7/static/styles/app_components.css#L524

Reply:
Sorry, I take that back. 

Because we use the `.alert` class for other alerts too.
So a better option will be to add `display: block`.